<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyndicateTools</class>
 <widget class="QDialog" name="SyndicateTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="checksumFrame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="filePath"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="browseFiles">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="md5Box">
          <property name="minimumSize">
           <size>
            <width>13</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>13</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="md5Label">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>MD5 :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="md5Check">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QPushButton" name="generateChecksums">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="shaBox">
          <property name="minimumSize">
           <size>
            <width>13</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>13</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shaLabel">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>13</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>13</height>
           </size>
          </property>
          <property name="text">
           <string>SHA1 :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shaCheck">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>13</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>13</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1" rowspan="2">
       <widget class="QPushButton" name="messageButton">
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="sha256Box">
          <property name="minimumSize">
           <size>
            <width>13</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>13</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sha256Label">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>SHA256 :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sha256Check">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>629</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
